go.mod

```diff
@@ -79,3 +79,5 @@ replace (
 	k8s.io/sample-cli-plugin => k8s.io/sample-cli-plugin v0.19.3
 	k8s.io/sample-controller => k8s.io/sample-controller v0.19.3
 )
+
+replace github.com/kube-object-storage/lib-bucket-provisioner => github.com/BlaineEXE/lib-bucket-provisioner 7027296f348eaeb7a254ef7c5ef8d8f0fb1cda03
```

Comment:
This is just temporary until https://github.com/kube-object-storage/lib-bucket-provisioner/pull/200 merges, right?

Reply:
Yes. Mostly I want to see that CI will run against this. I think if you have a look at https://github.com/kube-object-storage/lib-bucket-provisioner/pull/200 that would be most helpful right now. I've tested the upgrade scenario now with success.